Index: src/main/java/org/Game/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.Game;\n\nimport static org.Game.Helper.lang;\n\npublic class Main {\n    public static void main(String[] args) {\n        lang = \"Dansk\";\n        Deck deck = new Deck();\n        System.out.println(deck);\n\n        GameController gameController = new GameController();\n        gameController.startGame();\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/Game/Main.java b/src/main/java/org/Game/Main.java
--- a/src/main/java/org/Game/Main.java	(revision eb57100c01b34b8df7450b6e0a0043b73145563a)
+++ b/src/main/java/org/Game/Main.java	(date 1668157866040)
@@ -4,10 +4,6 @@
 
 public class Main {
     public static void main(String[] args) {
-        lang = "Dansk";
-        Deck deck = new Deck();
-        System.out.println(deck);
-
         GameController gameController = new GameController();
         gameController.startGame();
     }
